src/middlewared/middlewared/pytest/unit/test_schema.py

```diff
@@ -331,6 +331,14 @@ def dictargs(self, data):
         assert dictargs(self, value) == expected
 
 
+@pytest.mark.parametrize('items,value', [
+    ([List('b', items=[List('c', private=True)])], [[['a']]]),
+    ([Dict('b', Str('c', private=True))], [{'c': 'secret'}])
+])
+def test__schema_list_private_items(items, value):
+    assert List('a', items=items).dump(value) == '********'
```

Comment:
Can we also add a test with a List with multiple Dict as items with private attributes?

Or not worth it?

Reply:
Err, nevermind me, i am dumb, that what is happening here : )